boards/zigduino/Makefile.include

```diff
@@ -0,0 +1,9 @@
+# configure the terminal program
+PORT_LINUX  ?= /dev/ttyUSB0
+PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
+BAUD        ?= 57600
+
+ZIGDUINO_BOOTLOADER ?= atmegaboot
+BOOTLOADER ?= $(ZIGDUINO_BOOTLOADER)
```

Comment:
Not sure I understand the need for the `ZIGDUINO_BOOTLOADER` variable. The following would work the same:

```suggestion
BOOTLOADER ?=atmegaboot
```

And BOOTLOADER is still overridable.

Reply:
The advantage is that you can add this to your `~/.profile` or your `~/.bashrc` and it only applies for this specific board. E.g. I use the `optiboot` bootloader for all my Arduino Nano's instead of the default, but I still use the default bootloader on my Arduino Mega2560.

Reply:
updated according to #15887